shared/folders/files/render.desktop.js

```diff
@@ -49,13 +63,13 @@ const YouCanUnlock = ({youCanUnlock, isPrivate, backgroundMode}) => {
         youCanUnlock.map(device => (
           <ListItem
             key={device.name}
-            type='Large' action={device.onClickPaperkey
-              ? <Button label='Enter paper key' onClick={device.onClickPaperkey} type='Secondary' backgroundMode={backgroundMode} />
+            type='Large' action={device.type === 'backup'
```

Comment:
instead of closures in the store or in some map we just it down and call it appropriately